tests/schema_change_test.cc

```diff
@@ -708,7 +708,7 @@ SEASTAR_TEST_CASE(test_schema_digest_does_not_change_with_functions) {
     };
     return test_schema_digest_does_not_change_with_disabled_features(
         "./tests/sstables/schema_digest_with_functions_test",
-        std::set<sstring>{},
+        std::set<sstring>{"CDC"},
```

Comment:
I don't understand why a test called "test_schema_digest_does_not_change_with_functions", which apparently tests something with "functions" (UDF?), needs to list specifically CDC out of the gazillion other features we have... Is it because we have pre-CDC checksums saved, and now with CDC these saved checksums no longer match so we need to disable CDC? Is this the best we can do? Should a comment here (and in other places) explain this issue to the uninitiated like myself who would be reading this code?

Reply:
Yes. We have to either disable CDC feature on that test or update the checksums. Both ways are more or less equivalent.